CMakeLists.txt

```diff
@@ -367,24 +372,93 @@ else()
     message(STATUS "AES support enabled")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -maes")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -maes")
-  elseif(ARM)
+  elseif(ARM) #NB ARMv8 DOES support AES, but not yet coded
     message(STATUS "AES support disabled (not available on ARM)")
   else()
     message(STATUS "AES support disabled")
   endif()
 
+  if(ARM)
+  message(STATUS "Setting FPU Flags for ARM Processors")
```

Comment:
So...I guess that whole section is missing an indent. It would be nice if 'ARM' was highlighted like 'APPLE' and 'MINGW' and others to visually identify the section to the user.